lib/render-text.js

```diff
@@ -0,0 +1,8 @@
+import {element} from 'deku';
+
+export default text =>
+  text.map(({content, href}) =>
+    href
+    ? <a href={href}>{content}</a>
+    : content
+  );
```

Comment:
you can't bold & italic an embed text. These are currently used for Facebook & Twitter.